### PR TITLE
Home page

### LINKED ISSUE #23: Finish home page
Make it pretty

### PR DESCRIPTION
**Pull request by lvoytek**
It's the home page

Closes #23

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
+# Closed-Source Fonts that must be copied in
+DarlingModernRegular.woff
```
Comment (lboje) [inline client/.gitignore:4]: Do you just have this in the client root?

File: client/src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1 +1,35 @@
...
+		<div class="names">
+			<h1>Lena</h1>
+			<div class="ampersand">
+				<h1>&</h1>
+			</div>
+			<h1>Lucas</h1>
+		</div>
+
+		<div>
+		<div class="dateandlocationinfo">
```
Comment (lboje) [inline client/src/app/components/home/home.component.html:17]: Styling complaint >:(

hyphen separate words in class name in accordance with css convention pls

I am lodging this complaint here, but it applies to most of your classes.

Comment (lboje) [reply client/src/app/components/home/home.component.html:17]: technically we don't follow full convention anyway but whatever. in accordance with other class names

**Comment by lboje** [comment]
doesn't compile without darling :(

**Review by lboje — CHANGES_REQUESTED** [review]
doesn't compile without darling :(

File: client/.gitignore (hunk 1 above)
Comment (lvoytek) [reply client/.gitignore:4]: nah but I'll make it more clear

**Comment by lboje** [comment]
Overall, just looks a bit off-center?

image
A lot of deadspace between names and dates on mobile

Let's center this bit here
image

**Comment by lboje** [comment]
Other little comments -

I'd try and at least visually make Saturday and 6:00PM look more equal - maybe put a space between 6:00 and PM?
image

My real preference here would be to go the route of the invitation - make the lines equal width, and longer than the contents.

We're missing part of our address
image
What happened to 2655?

**Comment by lboje** [comment]
Ok ok ok ok only two more things right now

The first one is general - both mobile and desktop
I think it'd be good to add some top + bottom padding to the date/time lines
For instance I added this padding: 5px 0; line
image

and we go from this
image
to this
image

5 px looks good for larger views, but may be overkill for a phone? Try some other stuff. 0.25em looks pretty decent. 



The second thing is mobile only
straight up, we need a larger font size. You've kinda got something complicated going on with it right now, so I didn't wanna mess with it too much. 

But this is supposed to be what it looks like on a galaxy s20
image

I know you don't like media queries - but those are scaling down suuuuuuuper tiny :(

**Review by lboje — APPROVED** [review]

**Pull request merged**